### ISSUE TITLE
benchalerts: report how many commits ago was the last matching run on

### ISSUE THREAD

**Issue by austin3dickey** [conbench/conbench#969]
See https://github.com/conbench/conbench/pull/952#issuecomment-1485696348.

A high-signal, informative part of the `benchalerts` report would be to say how many commits ago the last matching run was run on. This would help users understand the context of whether there were any skipped commits.

**Comment by alistaire47** [comment]
Is this useful with a non-squash merge strategy? I don't think we want to benchmark every commit from a PR, and I'm not sure it matters how many commits were on a given PR. The number of merged PRs would be a better indicator, but I'm not sure how easy that is to get.

**Comment by austin3dickey** [comment]
>Is this useful with a non-squash merge strategy?

Maybe for merge-commits? Probably not rebase-commits. Related: #453. You're right that the number of merged PRs might be more useful.

>I don't think we want to benchmark every commit from a PR

Right, I forgot to mention: with the original implementation of this feature, we only gave this alert on the default-branch runs.

**Comment by alistaire47** [comment]
> > I don't think we want to benchmark every commit from a PR
> 
> Right, I forgot to mention: with the original implementation of this feature, we only gave this alert on the default-branch runs.

Yeah, I meant commits rebased from a PR to the default branch. But all this is hypothetical till we have a repo that needs it, so don't let me slow down getting something useful for now
